Source/Core/DolphinQt2/Main.cpp

```diff
@@ -68,6 +69,17 @@ int main(int argc, char* argv[])
   const optparse::Values& options = CommandLineParse::ParseArguments(parser.get(), argc, argv);
   const std::vector<std::string> args = parser->args();
 
+  std::unique_ptr<BootParameters> boot;
+  if (options.is_set("nand_title"))
+  {
+    const std::string hex_string = static_cast<const char*>(options.get("nand_title"));
+    if (hex_string.length() == 16)
+    {
+      const u64 title_id = std::stoull(hex_string, nullptr, 16);
+      boot = std::make_unique<BootParameters>(BootParameters::NANDTitle{title_id});
+    }
```

Comment:
consider printing an error on non-zero lengths that aren't valid - otherwise the option is silently ignored, which may be confusing. (in WX main, too)

Reply:
Done